src/porepy/models/constitutive_laws.py

```diff
@@ -3442,13 +3442,13 @@ class BartonBandis(pp.PorePyModel):
 
     The Barton-Bandis model represents a non-linear elastic deformation in the normal
     direction of a fracture. Specifically, the increase in normal opening,
-    :math:``\Delta u_n``, under a force :math:``\sigma_n`` (negative for compression),
+    :math:``\Delta u_n``, under a force :math:``t_n`` (negative for compression),
     is given as
 
     .. math::
 
         \Delta u_n = \Delta u_n^{max}
-            + \frac{\Delta u_n^{max} \sigma_n}{\Delta u_n^{max} K_n - \sigma_n}
```

Comment:
Could we use the somewhat less ambiguous "traction" or "contact_traction"?

Reply:
And as it is in math mode, maybe keeping symbol-like descriptions is better.